infra/resources/prod/main.tf

```diff
@@ -219,3 +219,18 @@ module "monitoring" {
   resource_group_name = azurerm_resource_group.itn_messages.name
   io_com_slack_email  = data.azurerm_key_vault_secret.io_com_slack_email.value
 }
+
+module "messages_eventhub" {
```

Comment:
please move this block into `eventhubs.tf` folder

can you also rename the folder into `_modules` to `eventhubs`?

Reply:
**please move this block into eventhubs.tf folde**r -> do you mean to create a infra/resources/prod/eventhubs.tf file ? in this case there is no problem

If you mean to move the block inside the eventhubs folder(after renaming infra/resources/_modules/messages_eventhubfolder into infra/resources/_modules/eventhubs) we lose all the shared locals and data that are inside the infra/resources/prod folder.

Make sense ?

Reply:
sorry, I meant file 🤦